### PR TITLE
feat(locale): add support for en-NP locale

### PR DESCRIPTION
**Pull request by rubek-joshi**
This adds support for Nepali locale which only differs slightly to the Indian one.

### CODE REVIEW

**Comment by mastermunj** [comment]
Hi @rubek-joshi, considering the locale en-NP is exactly same as en-IN, will you be able to use it?

**Comment by rubek-joshi** [comment]
> Hi @rubek-joshi, considering the locale en-NP is exactly same as en-IN, will you be able to use it?

It is very similar but NOT exactly the same. The key difference being en-NP doesn't use the term "Paise" and also the need for Arab and Kharab quantifiers. I could always override the en-IN in my projects but I prefer reducing boilerplate code so instead of having to do this:
```ts
const toWords = new ToWords({
  converterOptions: {
    currencyOptions: {
      name: 'Rupee',
      plural: 'Rupees',
      symbol: '₹',
      fractionalUnit: {
        name: 'Paisa',
        plural: 'Paisa',
        symbol: ''
      }
    }
  }
});
```
I prefer simply:
```ts
const toWords = new ToWords({ locale: "en-NP" });
```

Also, I need the Arab and Kharab quantifiers as well.

File: package-lock.json [hunk 1]
```diff
@@ -4090,6 +4089,9 @@
       },
       "engines": {
         "node": ">= 10"
+      },
+      "optionalDependencies": {
+        "@commitlint/load": ">6.1.1"
```
Comment (mastermunj) [inline package-lock.json:4094]: Please remove the changes from package-lock.

**Review by mastermunj — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (rubek-joshi) [reply package-lock.json]: Okay, done.

**Review by mastermunj — APPROVED** [review]

**Comment by mastermunj** [comment]
Thank you for your contribution @rubek-joshi. This is released with v3.5.0.

**Pull request merged**